packages/opentelemetry-exporter-collector/src/platform/node/CollectorExporterNodeBase.ts

```diff
@@ -95,13 +95,32 @@ export abstract class CollectorExporterNodeBase<
       this.logger.debug('Shutdown already started. Cannot send objects');
       return;
     }
-    if (this._protocol === CollectorProtocolNode.HTTP_JSON) {
-      sendWithJson(this, objects, onSuccess, onError);
-    } else if (this._protocol === CollectorProtocolNode.HTTP_PROTO) {
-      sendWithJsonProto(this, objects, onSuccess, onError);
-    } else {
-      sendWithGrpc(this, objects, onSuccess, onError);
-    }
+
+    const promise = new Promise(resolve => {
```

Comment:
same

Reply:
it's the same what here https://github.com/open-telemetry/opentelemetry-js/pull/1439/files/9cd8b6749b1cc91cfa0663308a4183fcf93bc70a#r472422313